src/ScriptEditor/ui/Options.ts

```diff
@@ -1,4 +1,10 @@
+import { editor } from "monaco-editor";
```

Comment:
Should be an `import type` since it is not being used for any runtime code (we've been terrible about this in most of the codebase). It's possible webpack is already optimizing this out of the runtime code anyway.